tests/BUILD

```diff
@@ -1,6 +1,11 @@
 cc_test(
     name = "simulator-test",
```

Comment:
nit (optional): from what I've seen, underscores are used in the bazel target name instead of hyphens

Reply:
Sounds good, I updated the names